package.json

```
@@ -23,6 +23,9 @@
 		"@digitalbazaar/ed25519-verification-key-2020": "^4.1.0",
 		"@digitalbazaar/vc": "^6.3.0",
 		"@wallet.storage/fetch-client": "^1.2.0",
+		"@digitalcredentials/ezcap": "^7.1.0",
+		"@digitalcredentials/ed25519-verification-key-2020": "^4.0.0",
```

Feel free to use `5.0.0-beta.2` -- that version has Typescript exports etc.
```suggestion
		"@digitalcredentials/ed25519-verification-key-2020": "^5.0.0-beta.2",
```